### PR TITLE
style: update `Content.getAll()` method return value

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

style: update `Content.delete()` method return value

### Motivation

Ditto.

### CODE REVIEW

File: files/en-us/web/api/contentindex/getall/index.md [hunk 1]
```diff
@@ -33,36 +33,43 @@ Returns a {{jsxref("Promise")}} that resolves with an {{jsxref('Array')}} of
...
+      - : A {{jsxref('String')}} description of the item. Used
+        in user-visible lists of content.
+    - `url`
+      - : A {{jsxref('String')}} containing the URL of the corresponding
+        HTML document. Needs to be under the scope of the current
+        {{domxref('ServiceWorker','service worker')}}.
+    - `category` {{Optional_Inline}}
+
+      - : A {{jsxref('String')}} defining the
+        category of content. Can be:
```
Comment (hamishwillee) [inline files/en-us/web/api/contentindex/getall/index.md:54]: Just some layout tidying

```suggestion
    - `id`
      - : A unique {{jsxref('String')}} identifier.
    - `title`
      - : A {{jsxref('String')}} title for the item.
        Used in user-visible lists of content.
    - `title`
      - : A {{jsxref('String')}} title of the item.
        Used in user-visible lists of content.
    - `description`
      - : A {{jsxref('String')}} description of the item.
        Used in user-visible lists of content.
    - `url`
      - : A {{jsxref('String')}} containing the URL of the corresponding HTML document.
        Needs to be under the scope of the current {{domxref('ServiceWorker','service worker')}}.
    - `category` {{Optional_Inline}}

      - : A {{jsxref('String')}} defining the category of content.
        Can be:
```

File: files/en-us/web/api/contentindex/getall/index.md [hunk 2]
```diff
@@ -33,36 +33,42 @@ Returns a {{jsxref("Promise")}} that resolves with an {{jsxref('Array')}} of
...
+        - `''` An empty {{jsxref('String')}}, this is the default.
+        - `homepage`
+        - `article`
+        - `video`
+        - `audio`
+
+    - `icons` {{Optional_Inline}}
+
+      - : An {{jsxref('Array')}} of image
+        resources, defined as an {{jsxref('Object')}} with the following data:
```
Comment (hamishwillee) [inline files/en-us/web/api/contentindex/getall/index.md:64]:
```suggestion
      - : An {{jsxref('Array')}} of image resources, defined as an {{jsxref('Object')}} with the following data:
```

**Review by hamishwillee — APPROVED** [review]
Hi @skyclouds2001 

Thanks very much. I think the only change in this was removal of duplicate `title` right? If this is not obvious from the files tab (it was not) next time can you add a note about what the change does in the description so reviewers can be sure they aren't missing something?

**Comment by skyclouds2001** [comment]
> Hi @skyclouds2001
> 
> Thanks very much. I think the only change in this was removal of duplicate `title` right? If this is not obvious from the files tab (it was not) next time can you add a note about what the change does in the description so reviewers can be sure they aren't missing something?

Thanks, I'll pay attention to that

**Pull request merged**